index.ts

```diff
@@ -1,11 +1,24 @@
-import { Client } from 'discord.js';
+import { Client, Intents } from 'discord.js';
 import { Config } from './Types';
 import './helpers/lifecycle';
 import * as bot_config from './config.json';
 
 export const config: Config = bot_config; // Just to shim it into shape and provide better types
 
-export const client = new Client();
+export const client = new Client({
+    intents: [
+        Intents.FLAGS.GUILDS,
+        Intents.FLAGS.GUILD_MEMBERS,
+        Intents.FLAGS.GUILD_BANS,
+        Intents.FLAGS.GUILD_PRESENCES,
+        Intents.FLAGS.GUILD_MESSAGES,
+        Intents.FLAGS.GUILD_MESSAGE_REACTIONS,
+        Intents.FLAGS.GUILD_MESSAGE_TYPING,
+        Intents.FLAGS.DIRECT_MESSAGES,
+        Intents.FLAGS.DIRECT_MESSAGE_REACTIONS,
+        Intents.FLAGS.DIRECT_MESSAGE_TYPING,
+    ],
+});
```

Comment:
Is this part of the upgrade?

Reply:
Yes https://discordjs.guide/additional-info/changes-in-v13.html#intents